packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -161,6 +189,7 @@
 			children = (
 				97C146EE1CF9000F007C117D /* Runner.app */,
 				680049172280D368006DD6AB /* image_picker_exampleTests.xctest */,
+				6801C8362555D726009DAF8D /* RunnerUITests.xctest */,
```

Comment:
This is that xctest bundle mentioned above, but shown in that same file explorer.